frontend/src/modules/search/components/SearchBox.js

```diff
@@ -239,6 +239,7 @@ export class SearchBoxBase extends React.Component<InternalProps, State> {
             return this.setState({ timeRange });
         }
 
+        // @ts-expect-error
```

Comment:
The variables on object keys are making TypeScript unhappy. I tried enum types a little bit on the keys, but that didn't work great, at least not in `TimeRangeFilter` below.